bindings/python/convert.py

```diff
@@ -145,7 +145,13 @@ def convert_multi(
         sf_filename = rename(pt_filename)
         sf_filename = os.path.join(folder, sf_filename)
         convert_file(pt_filename, sf_filename, discard_names=discard_names)
-        local_filenames.append(sf_filename)
+
+        if 'snapshots' in sf_filename.split(os.sep):
+            new_sf_path = os.path.join(folder, os.path.basename(sf_filename))
+            shutil.move(sf_filename, new_sf_path)
+            local_filenames.append(new_sf_path)
+        else:
+            local_filenames.append(sf_filename)
```

Comment:
```suggestion
```

Reply:
Let's split that into another PR if that's ok. The current detection seems a bit brittle and I'm not sure I understand what's the underlying issue.

Reply:
Actioned the reversal in another commit (we still need `local_filenames.append(sf_filename)`)